src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,79 @@
+.watch {
+  position: relative;
+  @include watch-size;
+
+  &__bg {
+    position: absolute;
+    @include watch-size;
+
+    background-color: blueviolet;
+
+    border: 45px solid burlywood;
+    border-radius: 50%;
+
+  }
+
+  &__seconds {
+    position: absolute;
+    @include second-size;
+    z-index: 3;
+
+    border-radius: $second-size / 5;
+    background-color: bisque;
+    border: 1px solid blanchedalmond;
+    left: $watch-size / 2;
+    top: $watch-size / 2;
+
+    animation-name: second-rotation;
+    animation-duration: 60s;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+    transform-origin: $second-width/4 3px;
+  }
+
+  &__minutes {
+    position: absolute;
+    @include minute-size;
+    z-index: 2;
+
+    border-radius: $minute-size / 5;
+    background-color: blue;
+    border: 1px solid blue;
+    left: $watch-size / 2;
+    top: $watch-size / 2;
+
+    animation-name: second-rotation;
+    animation-duration: 60s * 60;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
+    transform-origin: $minute-width/4 3px;
```

Comment:
it is a repeating animation script that could be combined into one common class, so the only difference between seconds and minutes is the duration, which can be specified by a modifier